graph_node_metrics.py

```diff
@@ -262,38 +320,68 @@ def collect_active_capacity_commitments_stats(client, provider_id, provider_name
         logger.error(f"Error collecting active capacity commitments: {e}")
         raise
 
-def collect_graph_networks_metrics(client):
+def collect_current_epoch_proof_stats(client, providers):
+
     try:
-        query = gql('''
-        query {
-            graphNetworks {
-                fltPrice
-                slashingRate
-                coreEpochDuration
-                capacityMaxFailedRatio
-                usdTargetRevenuePerEpoch
-                minRequiredProofsPerEpoch
-                dealsTotal
-                proofsTotal
-                offersTotal
-                providersRegisteredTotal
-                approvedProviders
-                effectorsTotal
-                capacityCommitmentsTotal
-                usdCollateralPerUnit
-                minRewardPerEpoch
-                maxRewardPerEpoch
-                vestingPeriodDuration
-                vestingPeriodCount
-                maxProofsPerEpoch
-                withdrawEpochsAfterFailed
-                difficulty
-            }
-        }
-        ''')
+        current_epoch = get_current_epoch(client)
+        network_info = get_network_info(client)
+
+        if current_epoch and network_info:
+            epoch_id = current_epoch['id']
+            max_proofs = int(network_info['graphNetworks'][0]['maxProofsPerEpoch'])
+            epoch_duration = int(network_info['graphNetworks'][0]['coreEpochDuration'])
+            time_from_epoch_start = int(time.time()) - int(current_epoch['startTimestamp'])
+
+            query = gql(f'''
+            query  {{
+              capacityCommitmentStatsPerEpoches(where: {{epoch: "{epoch_id}"}}) {{
+                submittedProofsCount
+                totalFailCount
+                id
+                activeUnitCount
+                capacityCommitment {{
+                  id
+                  provider {{
+                    id
+                    name
+                  }}
+                }}
+              }}
+            }}   
+            ''')
+            response = client.execute(query)
+            stats = response['capacityCommitmentStatsPerEpoches']
+            provider_ids = set([provider['id'] for provider in providers])
+
+            for stat in stats:
+                provider_id = stat['capacityCommitment']['provider']['id']
+                provider_name = stat['capacityCommitment']['provider']['name']
+                cc_id = stat['capacityCommitment']['id']
+
+                activeUnitCount = int(stat['activeUnitCount'])
+                if activeUnitCount > 0 and provider_id in provider_ids:
```

Comment:
if active unit count is 0 for an approved provider, that's a problem

maybe collect it as well?